apps/docs/content/guides/database/connecting-to-postgres.mdx

```diff
@@ -26,384 +20,121 @@ Supabase provides auto-updating Data APIs. These are the easiest way to get star
 
-## Connection pooler
+## Client libraries
 
-A "connection pool" is a system (external to Postgres) which manages Postgres connections.
+Supabase offers [client libraries](https://supabase.com/docs#client-libraries) for popular programming languages. These libraries provide a convenient wrapper around the Data APIs, making it easier to interact with your database using your preferred language.
 
-When a client makes a request, the pooler "allocates" an available connection to the client. When the client transaction or session is completed the connection is returned to the pool and is free to be used by another client.
+## Connecting to external libraries and tools
 
-<Image
-  alt="New migration files trigger migrations on the preview instance."
-  src={{
-    dark: '/docs/img/guides/database/connecting-to-postgres/how-connection-pooling-works.png',
-    light:
-      '/docs/img/guides/database/connecting-to-postgres/how-connection-pooling-works--light.png',
-  }}
-  caption="Connecting to the database directly vs using a Connection Pooler"
-  zoomable
-/>
+Beyond the [Supabase client-libraries](https://supabase.com/docs/guides/api/rest/client-libs), Supabase provides three connection strings for direct database access, compatible with all postgres libraries, such as [Prisma](/docs/guides/database/prisma) and [Drizzle](/docs/guides/database/drizzle), as well as tools like [PSQL](/docs/guides/database/psql) and [pgAdmin](/docs/guides/database/pgadmin).
 
-Every Supabase project comes with a connection pooler for managing connections to your database. The pooler provides 2 important services:
+You can find these connection strings in your [Database Settings](https://supabase.com/dashboard/project/_/settings/database).
 
-1. It manages connections for applications that connect and disconnect from the database frequently. For example, serverless functions and ORMs such as Prisma, Drizzle, and Kysely often make and drop connections to the database. If they connected directly each time, they would quickly exhaust your database server's memory. To connect to your database efficiently with such tools, you need a pooler.
-2. It provides an IPv4 endpoint to connect to your database, which you can use if your network provider doesn't support IPv6. This is unlike the direct database connection, which resolves to an IPv6 address, unless you've enabled the [IPv4 add-on](https://supabase.com/docs/guides/platform/ipv4-address).
-
-<Admonition type="tip">
+### Direct connections:
 
-The connection pooler is available in 2 modes: `Transaction` and `Session`.
+It connects directly to your Postgres instance. It is ideal for persistent servers (AWS EC2, Fly.io VMs, DigitalOcean Droplets...).
 
+<Admonition type="caution">
+  Direct connections use IPv6 addresses by default. If your environment isn't compatible or you're
+  unsure, you can check the [networking
+  guide](/docs/guides/platform/ipv4-address#supabase-and-ipv6-compatibility)
```

Comment:
nitpick - in one place, there's an absolute link to the docs with the hostname, while in another there is not